src/test/java/io/zipcoder/interfaces/StudentTest.java

```diff
@@ -0,0 +1,40 @@
+package io.zipcoder.interfaces;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StudentTest {
+	
+	@Test
+	public void testImplemenation(){
+		Student student = new Student("");
+		Assert.assertTrue(student instanceof Learner);
+	}
+	
+	@Test
+	public void testInheritance() {
+		Student student = new Student("");
+		Assert.assertTrue(student instanceof Person);
+		
+	}
+	
+	@Test 
+	public void testLearn() { 
+		
+		//if
+		double expectedNumberOfHours = 3;
+		Student someStudent = new Student("Richa");
+		
+		
+		//when
+		someStudent.learn(3);
```

Comment:
`someStudent.learn(expectedNumberOfHours);`
☝️ This statement expressed, "When some student learns the expected number of hours."